README.md

```diff
@@ -9,22 +9,39 @@ Funkcjonalności
         6. Lista zakupów, po wyjściu ze strony nie musi to być nigdzie zapisane, ewntualnie sesja, ciasteczka -> Search Grocery Products
         7. Kalkulator na przeliczanie miar gramy na kilogramy i takie tam. -> Convert Amounts
 ### Przykładowe uzycia api
-1. Search Recipes  
-a. https://api.spoonacular.com/recipes/complexSearch?query=pasta&maxFat=25&number=2&apiKey=1193248e5d2944ddbd9797097eaae839&apiKey=YOUR_API_KEY  
-b.  
-c.  
+1) Search Recipes  
+                a. https://api.spoonacular.com/recipes/complexSearch&apiKey=YOUR_API_KEY <-- To jest pusty template, do którego trzeba dodać co najmniej jeden parametr (bez parametrów nie działa). 
+                
+      b. W celu dodania parametru trzeba po 'complexSearch' wpisać '?', a następnie nazwę danego parametru i jego wartość po '='.
+https://api.spoonacular.com/recipes/complexSearch?query=rice&apiKey=YOUR_API_KEY  <-- To zapytanie do API zwróci 10 obiektów, które w parametrze 'query' mają słowo 'rice'.
```

Comment:
Where did you found information that this api call will return 10 objects?

Reply:
Well I made a mistake here. I should say "zwróci maksymalnie 10 obiektów" instead of  "zwróci 10 obiektów". The parameter "number" is set by default to 10 (it is not written in the docs, however in the "findByIngredients" it is written that the number is set by default to 10 and I also checked it by making some multiple weird "query" parameters like "mariuszpudzianowski" and in all cases i saw in the JSON file that "number" was set to 10)